### PR TITLE
Bump SPDX Licence ids to 0.2.0

### LINKED ISSUE #362: Update SPDX licence identifiers
```
# SPDX-License-Identifier: MIT
# OpenZeppelin Contracts for Cairo v0.1.0 (account/Account.cairo)
```
to
```
# SPDX-License-Identifier: MIT
# OpenZeppelin Contracts for Cairo v0.2.0 (account/Account.cairo)
```

### PR DESCRIPTION
**Pull request by martriay**
Fixes #362

### CODE REVIEW

**Review by JulissaDantes — APPROVED** [review]
Looks good!

**Review by andrew-fleming — COMMENTED** [review]
@martriay looks great! I think we're just missing a license id in the Account library and we need to bump up ReentrancyGuard

**Comment by martriay** [comment]
Great catches! I also renamed the reentrnacy guard file to match its licence

File: RELEASING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Releasing checklist:
 
 ```cairo
 # SPDX-License-Identifier: MIT
-# OpenZeppelin Contracts for Cairo v0.1.0 (account/Account.cairo)
+# OpenZeppelin Contracts for Cairo v0.2.0 (account/Account.cairo)
```
Comment (andrew-fleming) [inline RELEASING.md:11]: 👍

**Review by andrew-fleming — APPROVED** [review]
Good to go!

File: RELEASING.md (hunk 1 above)
Comment (martriay) [reply RELEASING.md:11]: haha NO! this should not have been there

```suggestion
# OpenZeppelin Contracts for Cairo v0.1.0 (account/Account.cairo)
```

**Pull request merged**